### PR TITLE
Bump security-scan image version in CIS-benchmark

### PR DESCRIPTION
**Pull request by doflamingo721**
Linked issue:
https://github.com/rancher/cis-operator/issues/149

### CODE REVIEW

**Review by dhruvmewada15 — DISMISSED** [review]
LGTM

**Comment by prachidamle** [comment]
Need to run `make remove` to delete earlier rc version

**Comment by doflamingo721** [comment]
@prachidamle I've removed the older RC version. Can you please review it again? Thanks.

**Comment by prachidamle** [comment]
@doflamingo721 did we run the `make remove` command - that would delete the rc1 and older assets. And we should add the new rc2 changes and assets and then run `make charts`

**Comment by doflamingo721** [comment]
@prachidamle Yes I had removed the rc1 assets and had added the rc2 changes, however I missed to update the version from rc1 to rc2. I have updated the versions now. Can you please review it again? Thanks.

**Review by prachidamle — DISMISSED** [review]
LGTM

File: release.yaml [hunk 1]
```diff
@@ -23,5 +23,7 @@ rancher-csp-adapter:
 - 0.1.0+up0.1.0-rc7
 rancher-cis-benchmark:
 - 2.0.5-rc1
+- 2.0.5-rc2
 rancher-cis-benchmark-crd:
 - 2.0.5-rc1
+- 2.0.5-rc2
```
[the remark this answers is not in the document]
Comment (doflamingo721) [reply release.yaml:25]: I can see only one version in release.yaml.

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ image:
     tag: v1.0.8
```
Comment (macedogm) [inline packages/rancher-cis-benchmark/charts/values.yaml:8]: @doflamingo721 can we also bump the `cis-operator` image to [v1.0.9](https://github.com/rancher/cis-operator/releases/tag/v1.0.9), please?
I did only an image refresh to retrieve updates and security patches. No additional modification was done.

**Review by macedogm — APPROVED** [review]
LGTM

File: release.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (prachidamle) [reply release.yaml:25]: yeah, this was a comment I had posted earlier - not sure why it showed up now. I dont have this issue anymore.

**Review by prachidamle — APPROVED** [review]

**Pull request merged**